Documentation/doc/Documentation/Usage.txt

```diff
@@ -162,6 +172,13 @@ if no debugging is intended. Users should thus run:
     cd CGAL-\cgalReleaseNumber/examples/Triangulation_2
     cmake -DCGAL_DIR=$HOME/CGAL-\cgalReleaseNumber -DCMAKE_BUILD_TYPE=Release . # we are here using a release tarball
 
+Note that for recent versions of Qt on brew (>= 5.14), the package has become "keg-only", which means it cannot be "linked" with brew.
+You will have to specify the Qt5_DIR by hand to cmake, using something like
+
+    -DQt5_DIR=/usr/local/opt/qt/lib/cmake/Qt5
+
+where '/usr/local/` is actually your current brew installation directory.
```

Comment:
The `qt5` formula seems keg-only since the commit https://github.com/Homebrew/homebrew-core/commit/ff3d0885f6aaf7ec8a0391d456b602b2269c1514, and at that time that was Qt 5.7.